### PR TITLE
Etoo la

### PR DESCRIPTION
**Pull request by idrice24**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,6 +44,9 @@
     "emailjs-com": "^2.6.4",
     "firebase": "^8.1.1",
     "jquery": "^3.5.1",
+    "moment": "^2.29.1",
+    "ngx": "^2.4.0",
+    "ngx-moment": "^5.0.0",
```
Comment (Ghislain1) [inline package.json:49]: Check the function from angular framework first

**Review by Ghislain1 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Ghislain1) [reply package.json:49]: Please dont use another library : **ANAGULAR should provide that feature** to format a  time /date

**Comment by Ghislain1** [comment]
read this page   we are going to talk about it this weekend

https://angular.io/api/common/DatePipe

**Comment by idrice24** [comment]
Ok but what I did, what us bad there or wrong on it??

-----Message d'origine-----
De : "Ghislain" <notifications@github.com>
Envoyé : ‎18/‎01/‎2021 16:52
À : "idrice24/mns" <mns@noreply.github.com>
Cc : "idrice24" <idrice.tsafouet@yahoo.com>; "Author" <author@noreply.github.com>
Objet : Re: [idrice24/mns] Etoo la (#428)

read this page we are going to talk about it this weekend
https://angular.io/api/common/DatePipe

**Comment by idrice24** [comment]
https://www.npmjs.com/package/ngx-moment
It is an angular library 

-----Message d'origine-----
De : "Ghislain" <notifications@github.com>
Envoyé : ‎18/‎01/‎2021 16:52
À : "idrice24/mns" <mns@noreply.github.com>
Cc : "idrice24" <idrice.tsafouet@yahoo.com>; "Author" <author@noreply.github.com>
Objet : Re: [idrice24/mns] Etoo la (#428)

read this page we are going to talk about it this weekend
https://angular.io/api/common/DatePipe

**Comment by idrice24** [comment]
I checked ngx-moment is the only want that can do what we are expecting !!

-----Message d'origine-----
De : "Ghislain" <notifications@github.com>
Envoyé : ‎18/‎01/‎2021 16:49
À : "idrice24/mns" <mns@noreply.github.com>
Cc : "idrice24" <idrice.tsafouet@yahoo.com>; "Author" <author@noreply.github.com>
Objet : Re: [idrice24/mns] Etoo la (#428)

@Ghislain1 requested changes on this pull request.



In package.json:
> @@ -45,6 +44,9 @@
     "emailjs-com": "^2.6.4",
     "firebase": "^8.1.1",
     "jquery": "^3.5.1",
+    "moment": "^2.29.1",
+    "ngx": "^2.4.0",
+    "ngx-moment": "^5.0.0",
Check the function from angular framework first

**Comment by Ghislain1** [comment]
Use angular legacy library . NOt externe library

**Comment by idrice24** [comment]
And if angular does not have what we want to do, what should I do??

-----Message d'origine-----
De : "Ghislain" <notifications@github.com>
Envoyé : ‎20/‎01/‎2021 14:26
À : "idrice24/mns" <mns@noreply.github.com>
Cc : "idrice24" <idrice.tsafouet@yahoo.com>; "Author" <author@noreply.github.com>
Objet : Re: [idrice24/mns] Etoo la (#428)

Use angular legacy library . NOt externe library

**Pull request closed without merging**